demisto_sdk/commands/init/initiator.py

```diff
@@ -109,6 +121,55 @@ def init(self):
             self.get_created_dir_name(created_object="pack")
             return self.pack_init()
 
+    @staticmethod
+    def convert_to_valid_name(name: str) -> str:
+        """Formats a (pack) name to a valid value
+
+        Specification:
+            A valid pack name does not contain any whitespace and may only contain alphanumeric, underscore, and
+            dash characters. The name must begin and end with an alphanumeric character. If it begins with an
+            alphabetical character, that character must be capitalized.
+
+        Behavior:
+            Individual words are titlecased, whitespace is stripped, and disallowed punctuation and space
+            characters are replaced with underscores.
+
+        Args:
+            name (str): The proposed pack name to convert to valid pack name format
+
+        Returns:
+            str: The reformatted pack name
+        """
+        temp = name.strip().strip('-_')
+        temp = capwords(temp)
```

Comment:
can we do that in the above line? not fan of assigning to the same variable running operations on